plotter/barchart_test.go

```diff
@@ -0,0 +1,178 @@
+// Copyright ©2015 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package plotter
+
+import (
+	"image/color"
+	"testing"
+
+	"github.com/gonum/plot"
+	"github.com/gonum/plot/vg"
+)
+
+func ExampleBarChart() {
+	// Create the plot values and labels.
+	values := Values{0.5, 10, 20, 30}
+	verticalLabels := []string{"A", "B", "C", "D"}
+	horizontalLabels := []string{"Label A", "Label B", "Label C", "Label D"}
+
+	// Create a vertical BarChart
+	p1, err := plot.New()
+	handleEx(err)
```

Comment:
I think that the motivation is to hide test-irrelevant logic, not to reduce the line count. But I agree with @kortschak that
```
if err := nil { log.Fatalf(err) }
```
isn't a lot of logic to be hidden.

I don't think that we need to cater to folks that will copy/paste this, and I'm not sure that `handleErr(err)` will fix the copy/paste problem anyway. Something like `handleErr` is not sufficient to drop in proper error handling; true error handling will need to do something within this function itself. For example, it will need to return the error or fix whatever caused it.

(BTW, do we still plan to fix plot.New so that it doesn't return an error, since there is no possible recovery for it?)